Cargo.toml

```diff
@@ -17,6 +17,8 @@ description = "A WIP implementation of the Cerberus attestation protocol"
 arrayvec = { version = "0.5.1", default_features = false }
 bitflags = "1.2.1"
 byteorder = { version = "1.3.4", default_features = false }
+clap = { version = "2.33.1", optional = true }
+serde_json = { version = "1.0", optional = true }
```

Comment:
Do you think you could separate these dependencies into another stanza?

Reply:
Done.